docs/Support/FAQ.md

```diff
@@ -161,3 +161,41 @@ ServiceTasks in Spiff Arena utilize a concept called Connector Proxy, an externa
 For tasks like checking if an API is functioning correctly, you can set up a Connector Proxy to handle the request.
 Detailed documentation available [here](https://spiff-arena.readthedocs.io/en/latest/DevOps_installation_integration/configure_connector_proxy.html).
 If you want to bypass security features of the restricted script engine and import modules from your script tasks, you can set the environment variable: `SPIFFWORKFLOW_BACKEND_USE_RESTRICTED_SCRIPT_ENGINE=false`
+
+
+**27: Where is the properties data stored in the properties panel?**
+
+**A:** The properties data is stored directly within the XML of the BPMN diagram. Some of this data is stored in extension elements. 
+For instance, the configuration for a service task can be found [here](https://github.com/sartography/sample-process-models/blob/sample-models-1/misc/jonjon/ham/ham.bpmn#L13) and instructions can be found [here](https://github.com/sartography/sample-process-models/blob/sample-models-1/misc/documentation/user-guide-basics/user-guide-basics.bpmn#L24). If you're considering bypassing the properties panel, it's essential to ensure that the XML output remains consistent with the expected format.
+
+**28: How do I start a task? What do I need besides BPMN?**
+
+**A:** To start a task, you'll need to have a proper BPMN diagram and a configured environment. The docker compose file, as mentioned on the [spiffworkflow.org](https://www.spiffworkflow.org/posts/articles/get_started/) website, provides a containerized environment for both the API and asynchronous processing. For a more robust production deployment, it's recommended to use separate containers for different functionalities.
+
+**29: Any documentation on how to set up our own openid provider?**
+
+**A:** If you're using the spiff-arena/spiffworkflow-backend, there's a script named `./keycloak/bin/start_keycloak` that can initiate a container serving as an example OpenID provider. This can be a good starting point if you're looking to set up your own OpenID provider.
+
+
+**30: What is the difference between SpiffWorkflow's core library and BPMN?**
+
+**A:** SpiffWorkflow is essentially divided into two main components. The core library is a versatile workflow library designed for a wide range of workflow patterns. On the other hand, the BPMN component was introduced later and is more aligned with the Camunda model. Our custom extensions, tailored for specific functionalities, reside in [SpiffWorkflow/spiff](https://github.com/sartography/SpiffWorkflow/tree/main/spiff).
+
+**31: Where can I configure an SMTP server for Spiffworkflow to send email notifications?**
+
+**A:** To configure an SMTP server for email notifications, you can utilize connectors and service tasks within SpiffWorkflow. For instance, connectors can be set up to send notifications to platforms like Slack. Alternatively, script tasks can also be employed for this purpose.
```

Comment:
I would probably remove the sentence about potentially using script tasks for this.

Reply:
Done